lib/ldp.js

```diff
@@ -17,6 +17,7 @@ var ldpContainer = require('./ldp-container')
 var parse = require('./utils').parse
 
 const DEFAULT_CONTENT_TYPE = 'text/turtle'
+const DEFAULT_EXTENSION = '.ttl'
```

Comment:
I think that `DEFAULT_CONTENT_TYPE` and `DEFAULT_EXTENSION` should be maintained in terms of a mapping from one to the other since they (in purpose) represent the same thing.

Reply:
You're right; this is going to get dropped though as per @timbl's comment.